src/xcode/ENA/ENA/Source/Services/HealthCertificate/__tests__/HealthCertificateServiceTests.swift

```diff
@@ -1895,6 +1893,7 @@ class HealthCertificateServiceTests: CWATestCase {
 		XCTAssertEqual(service.unseenNewsCount.value, 0)
 	}
 	
+	/*
```

Comment:
What about these tests?

Reply:
@pascal-brause  the unit tests have a separate ticket, I'm working on it at the moment and i UN-commented the the code "but the unit tests would not compile now until i finish the other ticket"

Reply:
okay, then it is fine for me